packaging/linux/flatpak/com.github.sunshinestream.sunshine.yml

```diff
@@ -60,6 +60,14 @@ modules:
         url: https://boostorg.jfrog.io/artifactory/main/release/1.79.0/source/boost_1_79_0.tar.bz2
         sha256: 475d589d51a7f8b3ba2ba4eda022b170e562ca3b760ee922c146b6c65856ef39
 
+  - name: curl
+    config-opts:
+      - --with-openssl
+    sources:
+      - type: archive
+        url: https://github.com/curl/curl/releases/download/curl-7_84_0/curl-7.84.0.tar.bz2
+        sha256: 702fb26e73190a3bd77071aa146f507b9817cc4dfce218d2ab87f00cd3bc059d
+
```

Comment:
I was informed that it is not necessary to build curl as `libcurl.so` is already included in the flatpak runtime.

Reply:
Excellent. I'll revert to last commit.